x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -16,6 +16,7 @@ export const APP_ICON = 'securityAnalyticsApp';
 export const APP_ICON_SOLUTION = 'logoSecurity';
 export const APP_PATH = `/app/security`;
 export const ADD_DATA_PATH = `/app/home#/tutorial_directory/security`;
+export const ADD_INTEGRATION_PATH = `/app/integrations/browse?category=security`;
```

Comment:
This path seems to be invalid. 

Suggestion: instead of hardcoding the path, use Fleet's path getter method which is exported out of the public folder:

```typescript
import {pagePathGetters} from '../path/to/fleet/plugin/public'
```

if wanting to land on the all integration page and search/filter by `security`:
```suggestion
export const ADD_INTEGRATION_PATH = pagePathGetters.integration_all({searchTerm: 'security'};
```

If wanting to land on the all integration page showing only the `security` category integrations:
```suggestion
export const ADD_INTEGRATION_PATH = pagePathGetters.integration_all({ category: 'security'};
```

and finally, if wanting to show only specific types of integrations on the Security category, you can combine the above:

```suggestion
export const ADD_INTEGRATION_PATH = pagePathGetters.integration_all({searchTerm: 'security', searchTerm: 'endpoint'};
```

Reply:
Thanks @paul-tavares , looks like the category version is what we want based on the decisions made in https://github.com/elastic/kibana/issues/107682

Reply:
I've moved this suggestion to the new PR: https://github.com/elastic/kibana/pull/112142#discussion_r710431477